mathsnuggets/widgets/taylor.py

```diff
@@ -0,0 +1,19 @@
+import sympy
+
+from mathsnuggets.core import fields, form
+
+test = {"function": "x sin x"}
+
+
+class Taylor(form.Form):
+    """Taylor Expansion"""
+
+    template = """Expand `expression``expand`"""
+
+    expression = fields.Expression("Expression")
+    x = fields.Expression("x", default="x")
+    order = 6
```

Comment:
`order = fields.Expression("Order", default=6)` so that this value can be changed

Reply:
And `x0 = fields.Expression("x0", default=0)`

Reply:
Will do